ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLChatAgentRunner.java

```diff
@@ -282,7 +282,13 @@ private void runReAct(
             .add(
                 ModelTensors
                     .builder()
-                    .mlModelTensors(Arrays.asList(ModelTensor.builder().name(MLAgentExecutor.MEMORY_ID).result(sessionId).build()))
+                    .mlModelTensors(
```

Comment:
Don't we need to add test for this?

Reply:
Let's add more test when merge to main branch. Merge to feature now to unblock the testing